### ISSUE TITLE
Calendar "icon" does not have sufficent color contrast

### ISSUE THREAD

**Issue by emmabastas** [Kompismoln/klimatkalendern#12]
<img width="151" height="148" alt="Image" src="https://github.com/user-attachments/assets/b9e1d684-6424-400a-a44b-6f770cfcff21" />

The purple on red does not have sufficient contrast to meet accessibility standars. I think we should just have text be black?

https://developer.mozilla.org/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast?utm_source=devtools&utm_medium=a11y-panel-checks-color-contrast
<img width="237" height="66" alt="Image" src="https://github.com/user-attachments/assets/c8bd3dfc-2fa3-4d16-8946-d9bc216dbf4b" />
